kubectl-fdb/cmd/configuration.go

```diff
@@ -34,11 +37,21 @@ func newConfigurationCmd(streams genericclioptions.IOStreams) *cobra.Command {
 		Short: "Get the configuration string based on the database configuration of the cluster spec.",
 		Long:  "Get the configuration string based on the database configuration of the cluster spec.",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			force, err := cmd.Root().Flags().GetBool("force")
```

Comment:
Can we use a term other than "force", I feel like that describes something that would be done to configuration command itself. Maybe change it to something like "--no-wait" or "--no-check"?

Reply:
I think that's a great idea and I would choose `--no-wait` or `--no-confirmation` to better represent what this flag actually does and nothing is forced 👍